CMakeLists.txt

```
@@ -6,6 +6,9 @@ list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake/modules)
 
 set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
+option(USE_BUNDLED_CTENSORFLOW
+  "Use the CTensorFlow module bundled in the toolchain" OFF)
```

Sorry, I'm reading this out of context and the meaning isn't fully clear to me. What is "the toolchain", in `Use the CTensorFlow module bundled in the toolchain`? Perhaps the wording can be edited to improve clarity.

---

Happy to reword it if you have ideas.  The toolchain is the toolchain image that you build from apple/swift's tensorflow branch.

---

We chatted offline. How about:
```suggestion
  "Use the CTensorFlow module bundled in the active Swift toolchain" OFF)
```

---

Updated after offline conversation